### ISSUE TITLE
API:s requiring credentials fail

### ISSUE THREAD

**Issue by shazmodan** [Zaid-Ajaj/Hawaii#28]
I have an API that requires a credentials cookie.

So when the OpenApiHttp.fs does this:

```fs
let sendAsync
    (method: HttpMethod)
    (basePath: string)
    (path: string)
    (extraHeaders: Header list)
    (parts: RequestPart list)
    : Async<int * string> =
    async {
        let requestPath = applyPathParts path parts

        let requestPathWithQuery =
            applyQueryStringParameters requestPath parts

        let fullPath =
            combineBasePath basePath requestPathWithQuery

        let! response =
            Http.request fullPath
            |> Http.method method
            |> applyJsonRequestBody parts
            |> applyMultipartFormData parts
            |> applyUrlEncodedFormData parts
            |> Http.headers extraHeaders
            |> Http.send

        let status = response.statusCode
        let content = response.responseText
        return status, content
    }
```

It doesn't include the credentials needed.
Maybe it can be fixed using Fable.SimpleHttp.Http.withCredentials? Like this:

```fs
let sendAsync
    (method: HttpMethod)
    (basePath: string)
    (path: string)
    (extraHeaders: Header list)
    (parts: RequestPart list)
    : Async<int * string> =
    async {
        let requestPath = applyPathParts path parts

        let requestPathWithQuery =
            applyQueryStringParameters requestPath parts

        let fullPath =
            combineBasePath basePath requestPathWithQuery

        let! response =
            Http.request fullPath
            |> Http.method method
            |> applyJsonRequestBody parts
            |> applyMultipartFormData parts
            |> applyUrlEncodedFormData parts
            |> Http.headers extraHeaders
            |> Http.withCredentials true // <------------- added this
            |> Http.send

        let status = response.statusCode
        let content = response.responseText
        return status, content
    }
```

**Comment by Zaid-Ajaj** [comment]
Hi @shazmodan, before I apply this fix, can you confirm that it actually works? You can edit the generated code and see if the error goes away

**Comment by shazmodan** [comment]
@Zaid-Ajaj I edited it manually and it solved my problem. There are however two functions affected as far as I know.

```fs
let sendAsync ...

let sendBinaryAsync ...
```

**Comment by Zaid-Ajaj** [comment]
Thanks for checking 🙏 applied this change and published it as of Hawaii v0.55 🚀
